.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@Lahphim @carryall
```

Comment:
```suggestion
@aetheryna @Lahphim @carryall
```

Reply:
Ah, didn't think I had to be included in it 😅 updated in https://github.com/aetheryna/google_scraper_ruby/pull/12/commits/c7199ca0f663bad636a030a80a1d7d43962a01c1